tests/test_chargebee_include_delete.py

```diff
@@ -0,0 +1,61 @@
+"""Test tap sync mode and metadata."""
+import re
+
+from tap_tester import runner, menagerie, connections
+
+from base import ChargebeeBaseTest
+
+
+class ChargebeeIncludeDeletedTest(ChargebeeBaseTest):
+    """Test tap sync mode and metadata conforms to standards."""
+
+    @staticmethod
+    def name():
+        return "tap_tester_chargebee_include_deleted_test"
+
+    def setUp(self):
+        self.include_deleted = None
+        super().setUp()
+
+    def get_properties(self):
+        properties = super().get_properties()
+
+        # include_deleted is an optional property for configuration
+        if self.include_deleted is False:
+            properties["include_deleted"] = self.include_deleted
+
+        return properties
+
+    def run_sync(self, expected_streams):
```

Comment:
These test cases are obsolete. Please update it as per the latest test cases added in the tap